File Upload Components and necessary files.

cszlo opened the pull request:
Styling to toggle between upload file and YoutubeUrl, Styling does not need to be permanent, just needed to get something together so I could see how it looked.

Commit:
File Upload Components and necessary files. Styling to toggle between upload file and YoutubeUrl.

Commit:
Cleaned up the styling a bit.

rareMatter committed:
Added a slide menu

Currently found under the fingerprint button, a slide menu that we can
fill with UI elements that can hide.

rareMatter committed:
Resolved conflicts

* Slide menu seems to be conflicting with something still, in the html
structure.

# Conflicts:
#	Visualizer-Web-App/src/app/app.component.html
#	Visualizer-Web-App/src/app/app.component.ts
#	Visualizer-Web-App/src/app/app.module.ts
#	Visualizer-Web-App/src/index.html

rareMatter committed:
Resolved slide menu conflict

Should be safe to add UI elements to the slide menu now, for easy
hiding.

cszlo commented on Visualizer-Web-App/src/app/app.component.html:
```diff
@@ -18,18 +18,20 @@
+        <li><a class="btn-floating btn-large waves-effect waves-green green pulse" onclick="Materialize.toast('Loved...', 3000, 'rounded')"><i class="material-icons">sentiment_very_satisfied</i></a></li>
+        <li><slide-menu></slide-menu></li>
+    </ul>
+</div>
```
@Avin12892 Not sure what the conflict was that you were having prior to this commit, but when I pull these changes the slide menu is acting funny. It's functioning fine, but it automatically slides out on the first time that I click the fingerprint button. Then I have to click the slide menu button twice to hide it. After that initial thing though it seems to work perfectly until I refresh. Are you experiencing this too?

rareMatter replied:
@cszlo I wasn't having an issue like that when I committed those changes. The conflicts were just that we had worked on the same files, I manually resolved them. But I am having some issues with weird slide menu behavior, after trying to change it a bit. I'll keep working on it.
Also, I think we should leave this pull request open until this branch is closer to finished or we'll end up just making lots pushes to master.

rareMatter committed:
Fixed slide menu behavior, temporary

The slide menu button being contained in the fingerprint button was
causing incorrect behavior.
I’ve moved it out for now, so we can continue adding elements to it.

rareMatter committed:
Added submit button and some styling

rareMatter committed:
UI foundation complete

Style updates, minor changes to slide menu layout.
Known bug: Placement of slide menu button is off on mobile.
